.gitignore

```diff
@@ -61,3 +61,6 @@ src/rmot.dll
 
 # Bugged directory from snapshot package
 tests/testthat/_snaps
+
+# Data prep files
+data-raw/
```

Comment:
Ok, this suggests you won't track this folder in git. So do you have good local copies?

Reply:
Yeah, they're all public data and I have Dropbox backups.